### ISSUE TITLE
Unique ID needed in ClosedLoopUtility and Log-File

### ISSUE THREAD

**Issue by carstenpaproth** [IvS-KULeuven/PlatoSim3#488]
We need a change in the ClosedLoopUtitility.cpp. The same unique identity shall be used for both sockets. We propose to use the process ID, see
[ClosedLoopChanges.txt](https://github.com/IvS-KULeuven/PlatoSim3/files/4851964/ClosedLoopChanges.txt)

Additionally this unique identity is needed as output in the log-file as well.

Could you implement these small changes/features soon, please?

**Comment by JorisDeRidder** [comment]
I implemented your suggestions, and ensured that the code compiles. 

I have not yet been able to properly test it, but I nevertheless pushed the changes so that you might help with this. Bertolt's test code does not use the standard `inputfile.yaml` configuration file, but uses 3 different dedicated yaml files: `inputfgs.yaml`, `onlineInput.yaml`, `offlineInput.yaml`. All three were outdated by now. I have now updated them, but in the long term we should get rid of these dedicated input files, and stick to `inputfile.yaml` which would ease the maintenance.  Even with the updated yaml files, the test code still doesn't run, because the windows are hardcoded but outdated (since the update of the reference frames). Fixing this will require a bit more time as it involves working into and rewriting part of Bertolt's scripts. Let me know if this is completely blocking you.
